OCIContainer/OCIContainer.cpp

```diff
@@ -666,6 +736,16 @@ const void OCIContainer::stateListener(int32_t descriptor, const std::string& na
     {
         __this->onContainerStopped(descriptor, name);
     }
+    #ifdef HIBERNATE_SUPPORT_ENABLED
+    else if (state == IDobbyProxyEvents::ContainerState::Hibernated)
+    {
+         __this->onContainerHibernated(descriptor, name);
```

Comment:
where is the onContainerHibernated and onContainerAwoken implementations ?

Reply:
Hi @goruklu ,
Missed to Implement.
Now implemented those API's.
Please review the changes.
Thanks.